controlsfx/src/main/java/org/controlsfx/property/editor/AbstractPropertyEditor.java

```diff
@@ -79,6 +79,17 @@ public AbstractPropertyEditor(Item property, C control) {
     public AbstractPropertyEditor(Item property, C control, boolean readonly) {
         this.control = control;
         this.property = property;
+
+        if (property.getObservableValue().isPresent()) {
+            property.getObservableValue().get().addListener((ObservableValue<? extends Object> o, Object oldValue, Object newValue) -> {
+                if (! suspendUpdate) {
+                    suspendUpdate = true;
+                    AbstractPropertyEditor.this.setValue((T) property.getValue());
+                    suspendUpdate = false;
+                }
+            });
+        }
+

```

Comment:
Just a code style suggestion, but otherwise fine. 
```suggestion
         property.getObservableValue().ifPresent(obs -> obs.addListener((ObservableValue<?> o, Object oldValue, Object newValue) -> {
            if (!suspendUpdate) {
                suspendUpdate = true;
                AbstractPropertyEditor.this.setValue((T) property.getValue());
                suspendUpdate = false;
            }
        }));

```

Reply:
Thanks for the suggestion, I applied it right now.